### PR TITLE
Feat/main view

### PR DESCRIPTION
**Pull request by Saerith**
Adds a main view for the console based menu-driven app for Urban Parks.

### CODE REVIEW

**Comment by vekk513** [comment]
Seems solid to me, were there any sections besides the job todo that need verified for logic?

File: mainView.java [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+		// show Park Manager options
+		if (myUser instanceof ParkManager) {
+			System.out.println("Hi, " + myUser.getFirstName() + "! What would you like to do?");
+			System.out.println(myInputPrompt);
+			System.out.println("1. Submit a new job");
+			String choice = scanner.nextChar();
+
+			switch(choice) {
+				case '1':
+					showSubmitNewJobMenu();
```
Comment (walkhh) [inline mainView.java]: No default case. If user enters 2 here the program not work or would end

File: mainView.java [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+	private static showSignupForJobMenu() {
+
+		System.out.println("Select a job from the list below to sign up for, input the corresponding number, and press enter:");
+		for(int i = JOB_LIST_START; i < JOB_LIST_END; i++) {
+			currJob = myJobCollection.get(i);
+			System.out.println(i + ": " + currJob.toString());
+		}
+
+		String choice = scanner.nextLine();
+		selectedJob = myJobCollection.getJob(Integer.parseInt(choice));
```
Comment (walkhh) [inline mainView.java]: Aren't jobs in jobCollection looked up by a hash/string?

**Comment by Saerith** [comment]
JFYI, this will not run until everyone else is done, so I have no way of knowing what hangups there will be. If your class has an issue, fix this view.

**Pull request merged**